composer.json

```diff
@@ -11,19 +11,19 @@
         }
     ],
     "require": {
-        "php": "^7.4 || ^8.0",
-        "symfony/options-resolver": "^5.4 || ^6.0 || ^7.0",
-        "psr/log": "^1.0 || ^2.0 || ^3.0"
+        "php": "^8.2",
+        "symfony/options-resolver": "^6.4 || ^7.1",
+        "psr/log": "^3.0"
     },
     "require-dev": {
         "doctrine/persistence": "^3.0",
         "doctrine/dbal": "^3.0 || ^4.0",
         "php-amqplib/php-amqplib": "^2.1 || ^3.0",
         "phpspec/prophecy": "^1.15",
         "phpspec/prophecy-phpunit": "^2.0",
-        "phpunit/phpunit": "^9.6 || ^10.0",
-        "symfony/error-handler": "^5.4 || ^6.0 || ^7.0",
-        "symfony/phpunit-bridge": "^5.4 || ^6.0 || ^7.0",
+        "phpunit/phpunit": "^11.5",
+        "symfony/error-handler": "^6.4 || ^7.1",
+        "symfony/phpunit-bridge": "^6.4 || ^7.1",
```

Comment:
using the `symfony/phpunit-bridge` with PHPUnit 11.5 does not make sense, given that we use it for its deprecation reporting feature (we should use the builtin feature in PHPUnit 11 instead)

Reply:
We could keep it to make it register the DebugClassLoader of `symfony/error-handler` to trigger its deprecations, but this requires using version 7.2+ of that component (and registering the extension in the phpunit.xml)